dbld/tarball

```diff
@@ -7,6 +7,7 @@ VERSION=`cat VERSION`
 ./autogen.sh
 
 cd /build
+rm -rf *
```

Comment:
As /build is retained, people might try save things there. Like [bash history](https://github.com/balabit/syslog-ng/pull/1932/commits/e836a7bd26e0f374ffb3092ae83a3fcf6624736d) :) (though in that specific example we are in luck that hidden directories are not deleted by rm -rf *). Better be safe, I suggest create a dedicated tarball directory or mktemp something

Reply:
Indeed. The separate directory is what zwa used to do, but I wanted to simplify. But you're right we shouldn't manage /build.